histomicstk/segmentation/label/__init__.py

```diff
@@ -12,6 +12,7 @@
 from .LabelPerimeter import LabelPerimeter
 from .ShuffleLabel import ShuffleLabel
 from .TraceBounds import TraceBounds
+from .trace_boundary import trace_boundary
```

Comment:
@cdeepakroy What do you think of how to compare the original ISBF algorithm in TraceBounds with the cython version of ISBF algorithm? I just added the same function on it to compare them.